pkg/jsonpath/v1alpha1/json_path.go

```diff
@@ -14,13 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package v1aplpha1 for creating aliases.
 package v1alpha1
 
 import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	. "github.com/openebs/maya/pkg/msg/v1alpha1"
+	"github.com/openebs/maya/pkg/msg/v1alpha1"
```

Comment:
You should not remove the `.` in front of `"github.com/openebs/maya/pkg/msg/v1alpha1"`
The dot is kept to import all the exported identifiers from `github.com/openebs/maya/pkg/msg/v1alpha1` to the `pkg/jsonpath/v1alpha1/json_path.go`'s scope.